core-data/index.js

```diff
@@ -10,12 +10,29 @@ import reducer from './reducer';
 import * as selectors from './selectors';
 import * as actions from './actions';
 import * as resolvers from './resolvers';
+import { default as entities, getMethodName } from './entities';
+
+const entityResolvers = entities.reduce( ( memo, { kind, name } ) => {
+	const methodName = getMethodName( kind, name );
+	return {
+		...memo,
+		[ methodName ]: ( state, key ) => resolvers.getEntityRecord( state, kind, name, key ),
+	};
+}, {} );
+
+const entitySelectors = entities.reduce( ( memo, { kind, name } ) => {
```

Comment:
My DRY senses are tingling 😄 

Maybe:

```js
const createEntityRecordGetter = ( source ) => entities.reduce( ( result, entity ) => {
	const { kind, name } = entity;
	const methodName = getMethodName( kind, name );
	result[ methodName ] = ( state, key ) => resolvers.getEntityRecord( state, kind, name, key );
	return result;
} );
```

Reply:
I think I saw a similar comment earlier today 😄

Reply:
Oof, the "Show outdated" toggles in GitHub are really easy to miss. 😬

Reply:
Since you both asked for it, I executed :)